src/runtime/contrib/tensorrt/tensorrt_runtime.cc

```diff
@@ -66,7 +78,16 @@ class TensorRTRuntime : public JSONRuntimeBase {
         use_implicit_batch_(true),
         max_workspace_size_(size_t(1) << 30),
         max_batch_size_(-1),
-        multi_engine_mode_(false) {}
+        multi_engine_mode_(false) {
+          const bool use_int8 = dmlc::GetEnv("TVM_TENSORRT_USE_INT8", false);
```

Comment:
I don't think use environment value is a perfect solution. I prefer making `use_int8` and `int8_calibration_batch` be an option of `tensorrt config` and writing is inside json. The api usage will be like this `mod, config = partition_for_tensorrt(mod, params, use_int8=True, int8_calibration_batches=10)`. cc @jcf94

Reply:
Ok, I will check them, and modify my codes

Reply:
I like the idea of having both options, with environment variable being able to override whatever is set during compilation.
However, I suggest we make this improvement in a separate PR, and just keep the environment variable method only in this PR.

Reply:
> I like the idea of having both options, with environment variable being able to override whatever is set during compilation.
> However, I suggest we make this improvement in a separate PR, and just keep the environment variable method only in this PR.

ok!

Reply:
Hi @FrozenGene @trevor-m @jcf94 I have modified original design, please review it. As for transfer parameters into partition_for_tensorrt api, like trevor suggested, probably I can provide a new and separate pr after this pr can be merged? Currently, I keep the design of setting up the environment variables

Reply:
> Hi @FrozenGene @trevor-m @jcf94 I have modified original design, please review it. As for transfer parameters into partition_for_tensorrt api, like trevor suggested, probably I can provide a new and separate pr after this pr can be merged? Currently, I keep the design of setting up the environment variables

OK.